src/CoreBundle/DependencyInjection/ChameleonSystemCoreExtension.php

```diff
@@ -174,7 +174,23 @@ private function addMailerConfig(array $mailerConfig, ContainerBuilder $containe
             return;
         }
         $mailerDefinition = $container->getDefinition('chameleon_system_core.mailer');
-        $mailerDefinition->addMethodCall('setSmtpHost', array($mailerConfig['host']));
+
+        $host = $mailerConfig['host'];
+        $port = (int) $mailerConfig['port'];
+        if (false !== ($portPosition = \mb_strrpos($host, ':'))) {
+            $portInHost = \mb_substr($host, $portPosition + 1);
+            if (true === \is_numeric($portPosition) && (25 === $port || $port === (int) $portInHost)) {
+                @trigger_error(
```

Comment:
We can't add deprecations in fix versions, only in minor versions. But do we even need to deprecate this? Maybe it makes sense to support both ways of setting the port (then we need to define precedence if both values are set).

Reply:
I can understand not wanting to remove deprecated code in a minor or patch version. I see no reason to not to deprecate code in a patch version.

Reply:
I also don't see any advantage supporting both configuration options. I think this only adds potential confusion - so I would like to remove that option as soon as possible.

In a later version we should really move the TLS option into a separate configuration parameter - I don't think it should be part of the host.

Perhaps we want to add that option at this point so we can keep the current behavior but deprecate it and remove it in the next version.

Reply:
Deprecations should only be introduced in minor versions. See https://semver.org/#how-should-i-handle-deprecating-functionality

Reply:
What @bestform says :-) - if we like to deprecated the port within the host, we need a separate PR targetting master.

Reply:
ok - I have moved the addition of the port parameter out of this PR. I think fixing the port handling in the host will be enough for the patch